cvat/schema.yml

```diff
@@ -9222,6 +9354,107 @@ components:
         compare_attributes:
           type: boolean
           description: Enables or disables annotation attribute comparison
+    RQJobDetails:
+      type: object
+      properties:
+        status:
+          allOf:
+          - $ref: '#/components/schemas/RQJobDetailsStatusEnum'
+          readOnly: true
+        message:
+          type: string
+          readOnly: true
+        id:
+          type: string
+          readOnly: true
+        operation:
+          $ref: '#/components/schemas/RQJobOperation'
+        percent:
+          type: number
+          format: double
+          readOnly: true
+          nullable: true
+        enqueue_date:
+          type: string
+          format: date-time
+          readOnly: true
+        start_date:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
+        finished_date:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
+        expire_date:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
+        owner:
+          allOf:
+          - $ref: '#/components/schemas/UserIdentifiers'
+          readOnly: true
+        result_url:
+          type: string
+          format: uri
+          readOnly: true
+          nullable: true
+      required:
+      - operation
+    RQJobDetailsStatusEnum:
+      enum:
+      - queued
+      - started
+      - failed
+      - finished
+      type: string
+      description: |-
+        * `queued` - queued
+        * `started` - started
+        * `failed` - failed
+        * `finished` - finished
+    RQJobOperation:
+      type: object
+      properties:
+        type:
+          type: string
+          readOnly: true
+        target:
+          allOf:
+          - $ref: '#/components/schemas/RQJobOperationTargetEnum'
+          readOnly: true
+        project_id:
+          type: integer
+          readOnly: true
+          nullable: true
+        task_id:
+          type: integer
+          readOnly: true
+          nullable: true
+        job_id:
+          type: integer
+          readOnly: true
+          nullable: true
+        format:
+          type: string
+          readOnly: true
+          nullable: true
+        name:
```

Comment:
What's this supposed to be?

Reply:
This param will be removed, it was presented in the original API design, but I was waiting for the client update to stop using it.